File: 10_3_2018/src/Bumper.java
import java.util.Scanner;

public class Bumper {
	
	public static void main(String[] args) {

		Scanner input = new Scanner(System.in);
		int x = input.nextInt();
		int y = input.nextInt();
		int z = input.nextInt();
		
		String a = "";
		for (int i = 1; i <= z; i++){
		    a = "";
		    if (i%x==0)
		    a+="Fizz";
		    if (i%y==0)
		    a+="Buzz";
		    if(a.length()==0) {
		    System.out.println(i);
		    continue;
		    }
		    System.out.println(a);
		  }
	}

}

File: 10_3_2018/bumper.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
  int x, y, z;
  cin >> x;
  cin >> y;
  cin >> z;

  string a;
  for (int i = 1; i <= z; i++){
    a = "";
    if (i%x==0)
    a+="Fizz";
    if (i%y==0)
    a+="Buzz";
    if(a.length()==0) {
    cout << i << endl;
    continue;
    }
    cout<<a<<endl;
  }

  return 0;
}
